metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/RedisInternalEnricher.java

```diff
@@ -228,8 +221,10 @@ private void loadEntity(EntityType entityType, MongoTerm term, Jedis jedis) {
         jedis.hset(entityType.cachedEntityPrefix + CACHED_URI, term.getCodeUri(),
             OBJECT_MAPPER.writeValueAsString(entityWrapper));
         List<String> parents = this.findParents(termList.getParent(), entityType.entityClass);
-        jedis.sadd(entityType.cachedEntityPrefix + CACHED_PARENT + term.getCodeUri(),
-            parents.toArray(new String[] {}));
+        if (parents != null && !parents.isEmpty()) {
```

Comment:
According to my sonarqube, parents can not be null here (it comes from a private method and it is not null).

Reply:
Correct! Removed null check.